source/features/closing-remarks.tsx

```diff
@@ -61,14 +73,14 @@ function addExistingTagLink(tagName: string): void {
 		);
 	}
 
+	const linkedTag = <a href={tagUrl} className="text-mono text-small">{tagName}</a>;
 	attachElement('#issue-comment-box', {
 		before: () => (
 			<TimelineItem>
 				{createBanner({
-					text: <>This pull request first appeared in <span className="text-mono text-small">{tagName}</span></>,
-					classes: ['flash-success'],
-					action: tagUrl,
-					buttonLabel: <><TagIcon/> See release</>,
+					icon: <TagIcon className="m-0"/>,
```

Comment:
The tag also aligns it well with `rgh-netiquette` in https://github.com/refined-github/sandbox/pull/1

<img width="1066" alt="Screenshot 11" src="https://user-images.githubusercontent.com/1402241/231422998-6bc01938-6f25-452d-9512-8c46d441e8c4.png">